lib/matplotlib/tests/test_preprocess_data.py

```diff
@@ -75,36 +58,6 @@ def func_no_ax_args(*args, **kwargs): pass
     with pytest.raises(AssertionError):
         _preprocess_data(label_namer="z")(func_args)
 
-    # but "ok-ish", if func has kwargs -> will show up at runtime :-(
```

Comment:
Deleted these as we don't support "implicit" label_name ("hoping that they'll show up in kwargs") anymore (this was basically present only to support `plot()` and `fill()` which don't have `y` explicitly in the signature but always have a `y` argument; this PR changes these functions to have their own parser).